backends/ent/backend.go

```diff
@@ -0,0 +1,90 @@
+// --------------------------------------------------------------
+// SPDX-FileCopyrightText: Copyright © 2024 The Protobom Authors
+// SPDX-FileType: SOURCE
+// SPDX-License-Identifier: Apache-2.0
+// --------------------------------------------------------------
+package ent
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"slices"
+
+	sqlite "github.com/glebarez/go-sqlite"
+	"github.com/protobom/protobom/pkg/storage"
+
+	"github.com/protobom/storage/internal/backends/ent"
+)
+
+// Backend implements the protobom.pkg.storage.Backend interface.
+type Backend struct {
+	// BackendOptions is the set of options common to all ent Backends.
+	*BackendOptions
```

Comment:
Don't embed the options here, add them as a property, this allows us to group the options apart from the objects properties:

```suggestion
	Options *BackendOptions
```

Reply:
Implemented suggestion